src/main/java/Server.java

```diff
@@ -10,15 +18,20 @@ public static void main(String[] args) throws InterruptedException {
     final Location location = Location.forGrpcInsecure("localhost", 4152);
 
     // Instantiate here the Producer you've created.
-    FlightProducer producer;
-
+    ServerFlightProducer serverFlightProducer = new ServerFlightProducer();
 
     //TODO Uncomment this code
-//    final FlightServer server = FlightServer.builder(allocator, location, producer).build().start();
-//    // Print out message for integration test script
-//    System.out.println("Server listening on localhost:" + server.getPort());
-//
-//    server.awaitTermination();
+    final FlightServer server;
+    try {
+      server = FlightServer.builder(allocator, location, serverFlightProducer)
+          .build()
+          .start();
+      // Print out message for integration test script
+      System.out.println("Server listening on localhost:" + server.getPort());
+      server.awaitTermination();
+    } catch (IOException e) {
+      e.printStackTrace();
```

Comment:
It's not a good practice to just ignore exceptions

Reply:
fixed